.github/workflows/merge-conflict-check.yml

```
@@ -0,0 +1,65 @@
+                  sleep 60  # Wait longer for rate limit
+                else
+                  echo "Failed to call GitHub API: $response"
+                  if [ $i -eq $max_retries ]; then
+                    echo "Maximum retries reached. Exiting."
+                    exit 1
+                  fi
+                  sleep $retry_delay
+                fi
+                continue
+              fi
+              
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)